tests/playbooks/roles/install-k3s/defaults/main.yaml

```diff
@@ -1,7 +1,7 @@
 ---
-k3s_release: v1.26.1+k3s1
+k3s_release: v1.28.4+k3s2
 worker_node_count: 1
-cluster_token: "9a08jv.c0izixklcxtmnze7"
+cluster_token: "K1039d1cf76d1f8b0e8b0d48e7c60d9c4a43c2e7a56de5d86f346f2288a2677f1d7::server:2acba4e60918c0e2d1f1d1a7c4e81e7b"
```

Comment:
I'm not a fan of this, but I assume we are not exposing k3s to the outside here.